mesa/mesaPROTON_OC.py

```diff
@@ -53,6 +53,11 @@ def __init__(self, seed=None):
         self.labour_status_by_age_and_sex = 0
         self.labour_status_range = 0
 
+        self.removed_fatherships = list()
```

Comment:
I don't get the whitepspacing here.. removed_fathership should go under some category (I'd keep it in the interventions area)